sway-interactive-screenshot

```diff
@@ -613,13 +618,14 @@ class NotificationAction(abc.ABC):
 
 
 class EditNotificationAction(NotificationAction):
-    name = "default"
+    name = "edit"
     description = "Edit"
 
     @classmethod
     def run(cls, *, filepath: Path, config_getter: Callable[[str], Any]) -> None:
         filepath_str = str(filepath.expanduser())
-        edit_capture(filepath_str)
+        command_replaced = config_getter("command").replace("{}", filepath_str);
+        subprocess.run(shlex.split(command_replaced), check=False)
```

Comment:
Expending `{}` should be done after tokenization otherwise this will break with spaces in the file path.

```suggestion
        cmd = [arg.replace("{}", filepath_str) for arg in shlex.split(config_getter("command"))]
        subprocess.run(shlex.split(command_replaced), check=False)
```

 We could also skip the first argument (the command name) but I don't think it really matter.